packages/twenty-server/src/modules/company/standard-objects/company.object-metadata.ts

```diff
@@ -50,7 +50,7 @@ export class CompanyObjectMetadata extends BaseObjectMetadata {
 
   @FieldMetadata({
     standardId: companyStandardFieldIds.address,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.ADDRESS,
```

Comment:
Is this a good idea? 🤔 This PR will require a data migration of a workspace anyway, right?

Reply:
@rostaklein `sync` and `health` command should handle this, but migration will be done manually I guess as we need to split addresses into multiples columns.
cc @charlesBochet, do we keep it as a text and change it later ?

Reply:
I think the best path for now is to introduce this as a completely new field and convert the existing address field into a custom field for existing workspaces (we don't have time to take care of the data migration ourselves, user can do it it). Having tooling to convert a standard field to a custom for existing workspaces will be useful (should we create a command?)

The additional issue here is that ideally we'd like to keep the same field name. Maybe rename "Address" to "AddressOld" while we're converting it to a custom field?

It's okay if we break the API

Reply:
Or should I keep the address field on company as text for now? 🤔